packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1979,6 +1983,10 @@ export default class Meeting extends StatelessWebexPlugin {
       this.wirelessShare = true;
     }
 
+    if (options.qualityOverride) {
```

Comment:
This needs to be added to the jsdoc typedef

Reply:
Addressed: https://github.com/webex/webex-js-sdk/pull/1660/files#diff-7362c0f286448790746a873a7c764a7bR1835

Reply:
Still not addressed